master/buildbot/interfaces.py

```diff
@@ -803,7 +803,7 @@ def getLogs():
         """Return a list of logs that describe the build as a whole. Some
         steps will contribute their logs, while others are are less important
         and will only be accessible through dictionary obtained from
-        py:meth:`getSteps`.  Each log is an object which implements the IStatusLog
+        get steps method.  Each log is an object which implements the IStatusLog
```

Comment:
no you need to put getSteps in https://github.com/buildbot/buildbot/blob/master/common/code_spelling_ignore_words.txt